src/website.rs

```diff
@@ -222,6 +231,77 @@ impl Benchmark {
             "cache_misses",
         ]
     }
+
+    fn calculate_means(timings: &[TimingRecord]) -> SimpleRuntimeMetrics {
+        let calc = |f: fn(&TimingRecord) -> f64| {
+            calculate_sample_mean(&timings.iter().map(f).collect::<Vec<_>>())
+        };
+        SimpleRuntimeMetrics {
+            nanoseconds: calc(|t| t.metrics.nanoseconds.median.raw()),
+            instructions: calc(|t| t.metrics.instructions.median.raw()),
+            context_switches: calc(|t| t.metrics.context_switches.median.raw()),
+            cpu_clock: calc(|t| t.metrics.cpu_clock.median.raw()),
+            branch_instructions: calc(|t| t.metrics.branch_instructions.median.raw()),
+            branch_misses: calc(|t| t.metrics.branch_misses.median.raw()),
+            cache_misses: calc(|t| t.metrics.cache_misses.median.raw()),
+            cache_references: calc(|t| t.metrics.cache_references.median.raw()),
+            cpu_cycles: calc(|t| t.metrics.cpu_cycles.median.raw()),
+        }
+    }
+
+    fn calculate_std_devs(
+        timings: &[TimingRecord],
+        means: &SimpleRuntimeMetrics,
+    ) -> SimpleRuntimeMetrics {
+        let calc = |f: fn(&TimingRecord) -> f64, mean: f64| {
+            calculate_sample_std_dev(&timings.iter().map(f).collect::<Vec<_>>(), mean)
```

Comment:
Do you think you could clean this closure up a bit in the same ways I suggested for `calculate_means`?